index.html

```diff
@@ -207,5 +207,53 @@ <h1>About me</h1>
             </ul>
         </div>
     </div>
+    <form action="https://formspree.io/f/mdovrwgl" method="post">
+        <span>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</span>
```

Comment:
- [ ] [Optional] 👍 Good job on the UI of your form! The form title looks visually appealing. However, there is room for improvement in terms of semantic structure. Instead of using a `<span>` tag to showcase the title, it is recommended to use a proper heading tag for better semantics and accessibility.

Using a heading tag, such as `<h2>` or `<h3>`, for your form title will not only improve the structure of your HTML but also provide clearer context to assistive technologies like screen readers.

Please consider updating the code and using an appropriate heading tag for the form title to ensure better semantic markup and enhance accessibility.

Keep up the great work! 👍✨